src/unitxt/stream_operators.py

```diff
@@ -155,5 +156,5 @@ class DuplicateSplit(MultiStreamOperator):
     def process(self, multi_stream: MultiStream) -> MultiStream:
         assert self.split in multi_stream
         generators = multi_stream
-        generators[self.to_split] = generators[self.split]
+        generators[self.to_split] = copy.deepcopy(generators[self.split])
```

Comment:
```suggestion
        stream_copy = copy.deepcopy(multi_stream[self.split])
        stream_copy.copying = True
        multi_stream[self.to_split] = stream_copy
        return multi_stream
```

Reply:
Explanation: when copying stream we need to make sure the instances are independent from the origin stream instances